syntax = "proto3";
package protos.playbooks;

import "google/protobuf/wrappers.proto";
import "protos/base.proto";


message PagerDutyIncidentEntryPoint {
    google.protobuf.StringValue pd_incident_id = 1;
    google.protobuf.StringValue pd_incident_title = 2;
    google.protobuf.StringValue pd_service_id = 3;
 }
